components/web/php/php-7_3/patches/02-jpeg.h.patch

```diff
@@ -0,0 +1,13 @@
+--- php-7.3.9/ext/gd/config.m4	2019-08-28 00:52:41.000000000 +0000
++++ php-7.3.9/ext/gd/config.m4	2019-10-16 20:40:20.850333627 +0000
+@@ -101,8 +101,8 @@ AC_DEFUN([PHP_GD_WEBP],[
+ AC_DEFUN([PHP_GD_JPEG],[
+   if test "$PHP_JPEG_DIR" != "no"; then
+ 
+-    for i in $PHP_JPEG_DIR /usr/local /usr; do
+-      test -f $i/include/jpeglib.h && GD_JPEG_DIR=$i && break
++    for i in $PHP_JPEG_DIR; do
```

Comment:
It seems that later code tries to use -L $PHP_JPEG_DIR/lib, which is incorrect (so not suitable for upstreaming), but likely it's fine enough for us.

Reply:
I take it as your approval :).

Reply:
OK